src/app/admin/usersTable/usersTableFilters.ts

```diff
@@ -67,20 +67,11 @@ export const buildUserRoleFilter = async (): Promise<BuildUserRoleFilterAndError
 
     const userRolesFromDb: UserRole[] = data.map((row) => row.role) ?? [];
 
-    const uniqueUserRoles = userRolesFromDb.reduce<UserRole[]>((accumulator, userRole) => {
-        if (!accumulator.includes(userRole)) {
-            accumulator.push(userRole);
-        }
-        return accumulator;
-    }, []);
-
-    uniqueUserRoles.sort();
-
     const userRoleFilter = checklistFilter<UserRow>({
         key: "userRole",
         filterLabel: "User Role",
-        itemLabelsAndKeys: uniqueUserRoles.map((userRole) => [userRole, userRole]),
-        initialCheckedKeys: uniqueUserRoles,
+        itemLabelsAndKeys: userRolesFromDb.map((userRole) => [userRole, userRole]),
+        initialCheckedKeys: userRolesFromDb,
```

Comment:
This fixes the bug described in the PR

Reply:
You're right it didn't actually fix it

I see two options
1. hardcode the roles in the dropdown 
2. update the roles in the dropdown automatically through a realtime channel

I tried to do the second option, but the problem was that, if we rebuild the filter every time the unique roles in the DB change, we lose the current state of the filter. To fix this, could a) change the way the filters work to be stateful we don't have to rebuild it every time, but this would be a lot of effort, or b) store the current state of the filter in the parent and pass that in to the function that build the filter whenever we need to rebuild it. This would work I think, but would add yet another state to the parent, when they shouldn't really be the one controlling the current state of the filter.

I've done option 1 and pushed it, but let me know if you want me to spend time on doing option 2

Reply:
yeah i think option 1 is okay. We'd definitely want to make it clear that it is tied to the database so that if we decide to change roles in the future, it'd be obvious enough to pick up (or even give an error)